src/themes.php

```diff
@@ -734,4 +734,15 @@
         "sideLabels" => "#C48519",
         "dates" => "#BA9D6F",
     ],
+    "deepBlue" => [
+        "background" => "#165795FF",
+        "border" => "#BA9D6F",
+        "stroke" => "#38DD69FF",
+        "ring" => "#37DD57FF",
+        "fire" => "#1CD577FF",
```

Comment:
Suggested fix based on previous commit (currStreakNum was removed)

```suggestion
        "fire" => "#1CD577FF",
        "currStreakNum" => "#639E29",
```